lib/repositories/language_repository.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:io';
+import 'dart:ui';
+import 'package:twake/models/globals/globals.dart';
+import 'package:twake/services/storage_service.dart';
+
+//TODO before the build, update the list of translated languages using weblate
+const List<String> translatedLanguages = ["en", "es"];
+
+class LanguageRepository {
+  final _storage = StorageService.instance;
+  LanguageRepository();
+
+  // To get a list of available languages from the device
+  // List<Locale> systemLocales = WidgetsBinding.instance!.window.locales;
+  // List<Locale> systemLocales = window.locales;
+
+  Locale get devlocale {
+    String deviceLanguage = Platform.localeName;
+
+    final String language = translatedLanguages.firstWhere(
+        (language) => language == deviceLanguage.replaceAll("_US", ""),
+        orElse: () => translatedLanguages[0]);
+
+    return Locale(language);
+  }
+
+  List<String> get languages {
```

Comment:
this is redundant because it already has a const public `translatedLanguages` above

Reply:
Agree